pagopa/io-app
# chore: [IOCOM-549] Display the `dueDate` in the transaction summary

## Short description
This PR enables the visualization of the `dueDate` in the transaction summary.

### Preview
| Transaction summary |
| --- |
| <img src="https://github.com/pagopa/io-app/assets/29163287/6ab0cdc2-922e-4ed4-90d3-f716215bf465" width="250" /> | 

## List of changes proposed in this pull request
- updated definitions
- enabled `TransactionSummaryRow` to display the `dueDate` in the transaction summary
- added the check on the status 504 for the `/payment-requests/{rptId}` and `/payment-activations` APIs

## How to test
1. checkout `io-dev-api-server` to this PR: https://github.com/pagopa/io-dev-api-server/pull/303
2. go to the details of the message
3. verify that the transaction summary contains the `dueDate`
